### DISCUSSION TITLE
Open PDF documents in a new tab/window

### DISCUSSION THREAD

**Discussion by clayjam** [microsoft-search/pnp-modern-search#1960] [Q&A] [answered]
When I perform a search and click a PDF document from the search results it opens in the same browser window and a user needs to navigate back using the browser. When a user navigates back the search terms entered into the search box are empty so the search results are different.

When clicking on a Word document it opens in a new browser tab. I am using the default Details List layout.

I looked into the handlebar expression and this already shows target='_blank'
![image](https://user-images.githubusercontent.com/29591956/160043424-531a0f4c-656a-476d-879a-417b4d2d3857.png)

How can we force PDF documents to open in a new tab/window ?

**Comment by wobba** [comment, marked as answer]
See https://docs.microsoft.com/en-us/sharepoint/dev/spfx/hyperlinking for how to solve this within SharePoint and SPFx.

**Reply by clayjam** [reply]
Thanks very much - worked perfectly.
